center/router/router_alert_cur_event.go

```diff
@@ -164,19 +189,21 @@ func (rt *Router) alertCurEventsList(c *gin.Context) {
 	}
 
 	ruleId := ginx.QueryInt64(c, "rid", 0)
+	gids, err := getUserGroupIds(c, rt, myGroups)
 
 	bgids, err := GetBusinessGroupIds(c, rt.Ctx, rt.Center.EventHistoryGroupView)
 	ginx.Dangerous(err)
 
 	total, err := models.AlertCurEventTotal(rt.Ctx, prods, bgids, stime, etime, severity, dsIds,
-		cates, ruleId, query)
+		cates, ruleId, query, gids, myGroups)
```

Comment:
Here, a single parameter `gids` will suffice, and later, you can determine whether to add it to the SQL query based on whether the length of `gids` is greater than 0